gp-includes/assets-loader.php

```diff
@@ -60,7 +60,7 @@ function gp_enqueue_styles( $handles ) {
  * @param string $handle A single style handle to enqueue.
  */
 function gp_enqueue_style( $handle ) {
-	if ( ! in_array( $handle, GP::$styles ) ) {
+	if ( false === in_array( $handle, GP::$styles ) ) {
```

Comment:
The coding standards check requires the comparison to be strict, hence the `false ===` instead of `!`.